test/unit/ngsiv2/amqpBinding-test.js

```diff
@@ -65,13 +65,11 @@ describe('AMQP Transport binding: measures', function() {
 
         nock.cleanAll();
 
-        // Note: /v1/updateContext response is not processed by IOTA so its content is irrelevant,
-        // as far as it is a 200 OK
         contextBrokerMock = nock('http://192.168.1.1:1026')
             .matchHeader('fiware-service', 'smartGondor')
             .matchHeader('fiware-servicepath', '/gardens')
-            .post('/v1/updateContext')
-            .reply(200, '{}');
+            .post('/v2/entities?options=upsert')
```

Comment:
Similar to what we have discussed in https://github.com/telefonicaid/iotagent-node-lib/pull/615#discussion_r198862320 maybe the usages of post in mock without payload in this PR should include a comment explaining the situation.

Reply:
Fixed with d6fa3bf1590ddd6dbbc2fe48aac48cae44516dc1